src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -143,7 +145,24 @@
         // returns map of sobject name => list of fields
         var allFieldsBySObject = this.groupFieldsBySObject(everyField);
 
-        Object.keys(allFieldsBySObject).forEach(function(sObjectName) {
+        const opportunitySObjectName = "Opportunity";
```

Comment:
We've tried to keep our quotations consistent where we can, and have settled on single quotes in our JS files for now. Could you make that change here to keep with that? Thanks!